tendermint-lite/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "tendermint-lite"
+version = "0.1.0"
+authors = ["Ethan Buchman <ethan@coinculture.info>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tendermint = { path = "../tendermint" }
```

Comment:
Shouldn't this be a proper dependency instead of a local path?

Reply:
You can do both...
```suggestion
tendermint = { version = "0.11", path = "../tendermint" }
```